mq.go

```diff
@@ -19,6 +19,11 @@ const (
 	// Describes states during reconnect.
 	statusReadyForReconnect int32 = 0
 	statusReconnecting      int32 = 1
+
+	ConnectionStateDisconnected int32 = 1
```

Comment:
It's make sense to add `ConnectionStateUnknown = 0` for the default state value.

Reply:
Or use ConnectionStateDisconnected as a default value.